.claude/skills/run-oloengine/driver.ps1

```
@@ -171,11 +211,26 @@ function Start-Editor {
+    param($proc, $timeoutSec, $exePath)
     $deadline = (Get-Date).AddSeconds($timeoutSec)
     while ((Get-Date) -lt $deadline) {
         if ($proc.HasExited) {
-            throw "Process exited early (exit code $($proc.ExitCode)) before a window appeared. Check OloEditor/OloEngine.log."
+            $msg = "Process exited early (exit code $($proc.ExitCode)) before a window appeared."
+            if ($proc.ExitCode -eq -1073741515) {
+                $msg += " Exit code 0xC0000135 = STATUS_DLL_NOT_FOUND: a required DLL is missing from the loader search path."
+            }
+            if ($exePath) {
+                $missing = $null
+                try { $missing = Get-UnresolvedDlls $exePath } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)